tools/build.csx

```diff
@@ -5,13 +5,13 @@
 
 using System.Net.Http;
 using System.Xml.Linq;
+using System.Runtime.CompilerServices;
 using static SimpleTargets;
 
 var solutionName = "FakeItEasy";
 
 var solution = "./" + solutionName + ".sln";
 var versionInfoFile = "./src/VersionInfo.cs";
-var packagesDirectory = Path.GetFullPath("packages");

```

Comment:
We no longer need this; all projects use the package cache in `%USERPROFILE%\.nuget\packages`.